.github/workflows/c-cpp.yml

```diff
@@ -8,13 +8,15 @@ on:
     branches: master
     paths:
     - '**'
+    - '!maps/**'
```

Comment:
These filters confound me sometimes.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths

Right now this job is being tested in the CI even with that filter. Maybe it will be more effective when it's on the master branch?